BraveRewardsUI/Extensions/DateExtensions.swift

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+extension Date {
+  /// Returns localized name of current month, using Gregorian calendar format.
+  var currentMonthName: String {
+    let calendar = Calendar(identifier: .gregorian)
+    let monthNumber = calendar.component(.month, from: self)
+    
+    // Due to declensions, Apple's `DateFormatter.monthSymbols` can't be used to get localized month name.
+    // In Polish language for example, a different word variation is used depending on date format.
+    
+    switch monthNumber {
```

Comment:
Can't we just use iOS's Calendar strings (and just apply `uppercased`): https://developer.apple.com/documentation/foundation/calendar/2293077-monthsymbols

Reply:
I thought we couldn't(see the comment lines above this line)
 but there's actually a `standaloneMonthSymbols` to help with this, fixing it